athena-neptune/athena-neptune.yaml

```diff
@@ -72,6 +75,8 @@ Resources:
           neptune_endpoint: !Ref NeptuneClusterEndpoint
           neptune_port: !Ref NeptunePort
           neptune_cluster_res_id: !Ref NeptuneClusterResourceID
+          iam_enabled: !Ref IAMEnabled
+          SERVICE_REGION: !Ref AWS::Region
```

Comment:
Why is this needed? Can't this be derived from the Lambda runtime env?

Reply:
SERVICE_REGION is needed by the SignV4 SDK SDK

https://docs.aws.amazon.com/neptune/latest/userguide/iam-auth-connecting-gremlin-java.html
https://github.com/aws/amazon-neptune-gremlin-java-sigv4/blob/master/src/main/java/com/amazon/neptune/gremlin/driver/sigv4/ChainedSigV4PropertiesProvider.java